sdk/tables/azure-data-tables/src/test/java/com/azure/data/tables/TablesAsyncClientTest.java

```diff
@@ -339,20 +361,27 @@ void updateEntityWithResponseAsync(UpdateMode mode) {
         createdEntity.getProperties().remove(oldPropertyKey);
         createdEntity.addProperty(newPropertyKey, "valueB");
 
-        // Act
-        StepVerifier.create(tableClient.updateEntityWithResponse(createdEntity, true, mode))
-            .assertNext(response -> assertEquals(expectedStatusCode, response.getStatusCode()))
-            .expectComplete()
-            .verify();
-
-        // Assert and verify that the new properties are in there.
-        StepVerifier.create(tableClient.getEntity(partitionKeyValue, rowKeyValue))
-            .assertNext(entity -> {
-                final Map<String, Object> properties = entity.getProperties();
-                assertTrue(properties.containsKey(newPropertyKey));
-                assertEquals(expectOldProperty, properties.containsKey(oldPropertyKey));
-            })
-            .verifyComplete();
+        // Act & Assert
+        if (mode == UpdateMode.MERGE && tableClient.getTableUrl().contains("cosmos.azure.com")) {
+            // This scenario is currently broken when using the CosmosDB Table API
+            StepVerifier.create(tableClient.updateEntityWithResponse(createdEntity, true, mode))
```

Comment:
Can we split this into another test case and disable it? I feel it'll get lost as a comment since there is no issue or TODO tracking this.

Reply:
Yes, once support is merged for multiple tests with the same name (#14801) I will separate these into two test classes, one for Cosmos and one for Storage, each with their own recordings.

Reply:
Tracking this work in #14930